src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/SmartContent.js

```diff
@@ -25,49 +25,114 @@ const filterCriteriaDefaults = {
 
 class SmartContent extends React.Component<Props> {
     smartContentStore: SmartContentStore;
-    filterCriteriaChangeDisposer: () => void;
+    filterCriteriaChangeDisposer: () => mixed;
 
     @computed get previousSmartContentStores() {
         return smartContentStorePool.findPreviousStores(this.smartContentStore);
     }
 
-    constructor(props: Props) {
-        super(props);
-
+    @computed get presentations(): Array<Presentation> {
         const {
-            formInspector,
             schemaOptions: {
-                exclude_duplicates: {
-                    value: excludeDuplicates = false,
+                present_as: {
+                    value: schemaPresentations = [],
                 } = {},
+            } = {},
+        } = this.props;
+
+        if (!Array.isArray(schemaPresentations)) {
+            throw new Error(
+                'The "present_as" schemaOption must be a string, but received ' + typeof schemaPresentations + '!'
```

Comment:
i guess this should be `must be an array`

Reply:
right :slightly_smiling_face: